Corrections.py

```diff
@@ -193,12 +193,12 @@ def getNormalisationCorrections(df, config, sample, ana_cache=None, return_varia
             df = df.Define(weight_rel_name, f'static_cast<float>({weight_name}/weight_TauID_{central})')
             all_weights.append(weight_out_name)
     if mu!= None:
-        df, muID_SF_branches = mu.getMuonIDSF(df,isCentral)
+        df, muID_SF_branches = mu.getMuonIDSF(df, nLegs, isCentral)
         all_weights.extend(muID_SF_branches)
     if ele!= None:
-        df, eleID_SF_branches = ele.getIDSF(df, isCentral)
+        df, eleID_SF_branches = ele.getIDSF(df, nLegs, isCentral)
         all_weights.extend(eleID_SF_branches)
-    if puJetID!=None:
+    if puJetID!=None and nLegs == 2:
```

Comment:
Because the current `puJetID` uses `b1` & `b2` branches. This is a temporary workaround while waiting for #23 to be merged (where jet PU ID weights are computed for all jets).